autopilot/accounts.go

```diff
@@ -157,21 +157,15 @@ func (a *accounts) refillWorkerAccounts(w Worker) {
 		if a.markRefillInProgress(workerID, c.HostKey) {
 			go func(contract api.ContractMetadata, l *zap.SugaredLogger) {
 				rCtx, cancel := context.WithTimeout(ctx, time.Minute)
-				if account, err := refillWorkerAccount(rCtx, a.a, w, workerID, contract, l); err == nil {
-					a.l.Infow("Successfully funded account",
-						"account", account,
-						"host", contract.HostKey,
-						"balance", maxBalance,
-					)
-				}
+				_ = refillWorkerAccount(rCtx, a.a, w, workerID, contract, l)
```

Comment:
why get rid of the logging?

Reply:
oh I see what you did, I did this on purpose, I wanted to log successes in any case, and not log failure in some cases